Source/WebCore/rendering/TextBoxPainter.cpp

```diff
@@ -95,7 +95,7 @@ InlineIterator::TextBoxIterator TextBoxPainter<TextBoxPath>::makeIterator() cons
 }
 
 template<typename TextBoxPath>
-void TextBoxPainter<TextBoxPath>::paint()
+void TextBoxPainter<TextBoxPath>::paint(const RenderStyle& renderStyle)
```

Comment:
Style is already available as a member. I don't see why it is also added as a parameter.

Reply:
No reason indeed, it's fixed.